packages/ckeditor5-autosave/src/autosave.js

```diff
@@ -213,22 +224,8 @@ export default class Autosave extends Plugin {
 	 * @private
 	 */
 	_save() {
-		const version = this.editor.model.document.version;
-
-		// Change may not produce an operation, so the document's version
-		// can be the same after that change.
-		if (
-			version < this._lastDocumentVersion ||
-			this.editor.state === 'initializing'
```

Comment:
This is wrong assumption. Data change might be triggered during editor initialization (when editor state is `initializing`) but `_save()` might be actually called later (with delay) when the editor state is already `ready`.